package.json

```diff
@@ -16,25 +16,20 @@
     "qdequele <quentin@meilisearch.com>"
   ],
   "license": "MIT",
-  "main": "./dist/bundles/meilisearch.cjs",
-  "module": "./dist/bundles/meilisearch.mjs",
-  "typings": "./dist/types/index.d.ts",
-  "types": "./dist/types/index.d.ts",
-  "jsdelivr": "./dist/bundles/meilisearch.umd.js",
-  "unpkg": "./dist/bundles/meilisearch.umd.js",
```

Comment:
Don't we risk breaking compatibility by removing this? 

[UNPKG](https://www.unpkg.com/) docs reads:
> If you omit the file path (i.e. use a “bare” URL), unpkg will serve the file specified by the unpkg field in package.json, or fall back to main.

I'm not sure what that means since we don't have a `main` field anymore.

Reply:
Alright, I'm changing `"jsdelivr"` to `"main"`.
According to this, it should still work: https://www.jsdelivr.com/documentation#id-publishing-packages

Reply:
I've already re-added the `"main"` field. For the currently supported CDNs this suffices.

What CDNs do we want to support? Right now [esm.sh](https://esm.sh/), [jsDelivr](https://www.jsdelivr.com/), [Skypack](https://www.skypack.dev) and [UNPKG](https://unpkg.com/) seems to work, but honestly we shouldn't strive to support any specific one. If they need a very specific `package.json` setup, it's not worth the hassle and "pollution" of `package.json` with arbitrary properties left and right.